libraries/AC_Avoidance/AC_Avoid.h

```diff
@@ -87,6 +92,9 @@ class AC_Avoid {
     // return true if limiting is active
     bool limits_active() const {return (AP_HAL::millis() - _last_limit_time) < AC_AVOID_ACTIVE_LIMIT_TIMEOUT_MS;};
 
+    // Enable or disable active proximity avoidance
+    void set_active_proximity(bool state) {_active_proximity = state;};
```

Comment:
I think maybe we should call the feature "backup" or "backaway" instead of "active".  It's your baby so I won't press on this too much but I think for example that the simple avoidance we do in alt-hold is very "active".  that's obviously a completely different aspect of simple avoidance but I think it could be confused with this new feature so I think a more specific name would be good.